test/src/org/labkey/test/tests/targetedms/TargetedMSUtilizationCalendarTest.java

```diff
@@ -20,7 +20,7 @@ public class TargetedMSUtilizationCalendarTest extends TargetedMSTest
     @Override
     protected @Nullable String getProjectName()
     {
-        return getCurrentTestClass().getSimpleName() + " Project";
+        return TargetedMSUtilizationCalendarTest.class.getSimpleName() + " Project";
     }
```

Comment:
Curious, how is it different?

Reply:
In reality, it probably doesn't ever evaluate to something different but `getCurrentTestClass` is a static method in BaseWebDriverTest that could return `null`.
`getClass()` is actually the method that this should have used (I just changed it).